docs/rest/ReindexFlowResourceBase.http

```diff
@@ -0,0 +1,170 @@
+  },
+  "status": "final",
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code": "laboratory",
+          "display": "Laboratory"
+        }
+      ],
+      "text": "Laboratory"
+    }
+  ],
+  "code": {
+    "coding": [
+      {
+        "system": "http://loinc.org",
+        "code": "34532-2",
+        "display": " Blood type and Indirect antibody screen panel - Blood"
+      }
+    ],
+    "text": "Blood Group Panel"
+  },
+  "subject": {
+    "reference": "Patient/infant"
+  },
+  "effectiveDateTime": "2018-03-11T16:07:54+00:00",
+  "hasMember": [
+    {
+      "reference": "Observation/bloodgroup"
+    },
+    {
+      "reference": "Observation/rhstatus"
+    }
+  ]
+}
+
+###
+# Create a new search parameter that will return the resource ID.
+# This will have the same behaviour as the existing search parameter, "_id".
+POST {{baseUrl}}/SearchParameter HTTP/1.1
+content-type: {{contentType}}
+
+{
+        "resourceType": "SearchParameter",
+        "id": "Resource-id",
+        "url": "http://hl7.org/fhir/SearchParameter/Resource-test-id",
+        "name": "test-id-name",
+        "status": "active",
+        "experimental": false,
+        "date": "2019-10-24T11:53:00+11:00",
+        "publisher": "Health Level Seven International (FHIR Infrastructure)",
+        "contact": [
+          {
+            "telecom": [
+              {
+                "system": "url",
+                "value": "http://hl7.org/fhir"
+              }
+            ]
+          },
+          {
+            "telecom": [
+              {
+                "system": "url",
+                "value": "http://www.hl7.org/Special/committees/fiwg/index.cfm"
+              }
+            ]
+          }
+        ],
+        "code": "test-id-code",
+        "base": [
+          "Resource"
+        ],
+        "type": "token",
+        "description": "Logical id of this artifact",
+        "expression": "Resource.id",
+        "xpath": "f:Resource/f:id",
+        "xpathUsage": "normal"
+}
+
+###
+# Trigger a reindexing operation.
+# Copy the response's Content-Location header value for the next request.
+# You can paste it here for now: 34dbddb7-6b02-46d3-9627-095b8afeed25
+POST {{baseUrl}}/$reindex HTTP/1.1
+content-type: {{contentType}}
+
+{  "resourceType": "Parameters", "parameter": [] }
+
+###
+# Check the status of the reindexing operation (scroll down in the response body to see the status).
+# Make sure the ID used here is the one pulled from the last request's response.
+GET {{baseUrl}}/_operations/reindex/34dbddb7-6b02-46d3-9627-095b8afeed25 HTTP/1.1
+
+###
+# Replace the ID with the value retrieved when creating the patient.
+# This should return a single patient.
+GET {{baseUrl}}/Patient?test-id-code=e181cab2-9c1d-4025-916d-d63786ccbdcb&_summary=count HTTP/1.1
```

Comment:
This works as intended, but I find it strange that _summary=count doesn't return a self link. For this test a self link would show more proof that the search parameter was honored. I also don't think it lines up with the spec's intention for _summary. I don't feel it is meant to summarize the bundle, just the resources in the bundle.

Reply:
I agree that having the self link would be helpful for testing. Replaced `_summary=true` with `_total=accurate` in 907bcfa.

I created an investigate story to examine the behaviour of `_summary` (please see [AB#80540](https://microsofthealth.visualstudio.com/Health/_workitems/edit/80540))

Reply:
I would argue what `_summary` behavior is correct, at least from spec perspective.